src/test/java/bio/terra/service/snapshotbuilder/query/filtervariable/SimpleFilterVariableForTests.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.service.snapshotbuilder.query.filtervariable;
+
+import bio.terra.service.snapshotbuilder.query.FilterVariable;
+import bio.terra.service.snapshotbuilder.query.TableVariable;
+import java.util.List;
+
+public class SimpleFilterVariableForTests implements FilterVariable {
```

Comment:
Sonar Cloud recommends a new test class for this new class

Reply:
I was thinking about that. Honestly, I'm very hesitant because this class is only used for tests, so if it breaks it is very low impact, while adding tests would make it higher effort to make changes as we need new functionality for filter variables for our tests.

Reply:
I think we can revert the change to add `getTables()`, and then this could be removed.

But since it's in the `test` hierarchy I don't know why sonar would expect to see a test for it.